www/pages/cloudflare-workers-deployment.md

```diff
@@ -0,0 +1,56 @@
+## Cloudflare Workers Deployment
+
+[Cloudflare Workers](https://workers.cloudflare.com/) is an excellent option as a CDN for deploying your Greenwood site, though if your are new to using this service consider deploying with [Netlify](https://www.netlify.com) as it is a simpler process.
```

Comment:
> though if your are new
I think this might mean to be _"though if **you are** new"_

Reply:
weird, my local has 'you are' not 'your'... good catch.